sdk/python/flet/page.py

```diff
@@ -294,6 +300,7 @@ def go(self, route):
             )
         )
         self.update()
+        self.query() # Update query url
```

Comment:
not sure I understand this line. Could you please elaborate?

Reply:
Well, as I mentioned in previous comment, `self.query()` is using `__call__` magic method to keep updating url every time `go` method is called, but I'm thinking updating url, using `route_change` event.

Reply:
ah, magic! :)